content/tv/code/0064/index.md

```diff
@@ -0,0 +1,21 @@
+---
+Date: '2022-04-06T15:00:00-05:00'
+Description: "Spring Tips Live with Josh Long"
+PublishDate: '2022-04-04T10:00:00-05:00'
+aliases:
+- /tv/code/67
+episode: '67'
+image: /tv/code/images/dan-vega.jpg
+explicit: 'no'
+hosts:
+- Dan Vega
+lastmod: '2022-04-04'
+minutes: 120
+title: "Spring for GrqphQL"
```

Comment:
```suggestion
title: "Spring for GraphQL"
```

Reply:
@danvega Looks like this is why it was failing, there was a typo in the title. Once this change is accepted, it should pass and I can merge :)